diff --git a/dist/index.js b/dist/index.js
index 057c2897fdb1a01960724b83707a9c9d28205988..86718db73cce638941eb5f79f8c83bac93a17401 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -267,7 +267,7 @@ var codePlugin = (md, {
     const info = token.info ? md.utils.unescapeAll(token.info).trim() : "";
     const language = resolveLanguage(info);
     const languageClass = `${options.langPrefix}${language.name}`;
-    const code = options.highlight?.(token.content, language.name, "") || md.utils.escapeHtml(token.content);
+    const code = options.highlight?.(token.content, language.name, info || "") || md.utils.escapeHtml(token.content);
     token.attrJoin("class", languageClass);
     let result = code.startsWith("<pre") ? code : `<pre${slf.renderAttrs(token)}><code>${code}</code></pre>`;
     const useVPre = resolveVPre(info) ?? vPreBlock;
